packages/playground/src/components/execution/step-execution.tsx

```diff
@@ -68,6 +70,12 @@ export const StepExecution: React.FC<StageExecutionProps> = ({ step, stepIndex }
               <ConsoleLogs logs={data[stepIndex]} />
             </ScrollArea>
           </TabsContent>
+          <TabsContent value={StepExecutionTab.INPUT}>
+            {/*here is the execution details of input table */}
+            <ScrollArea className="h-[calc(100vh-23rem)] border-t pt-4">
```

Comment:
there is a ScrollArea inside the table component as well.. why do we need two nested scroll areas? I think the other one should be removed.

Reply:
@jessieyu777 I think you might have referred the implementation for `LOGS` tab which currently has this height set since it doesn't handle scroll on it's on today.

Reply:
We confirmed that the table from `canary` doesn't handle scroll. (no `ScrollArea` component in it).

Reply:
We still need to handle the calculation properly, this applies to all the tabs for Step Execution.